MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -899,23 +898,25 @@ private void BuildStreamVideoItem(
             string? audioCodec)
         {
             // Prefer matching video codecs
-            var videoCodecs = ContainerProfile.SplitValue(videoCodec);
+            List<string> videoCodecs = [];
+            if (audioCodec is not null)
+            {
+                videoCodecs.AddRange(ContainerHelper.Split(videoCodec));
+            }
```

Comment:
Btw, is this work? But we probably lose the benefit of the `List'.
```suggestion
            List<string> videoCodecs = ContainerHelper.Split(videoCodec);
```